gitlabform/gitlab/resource_group.py

```diff
@@ -0,0 +1,22 @@
+from gitlabform.gitlab.projects import GitLabProjects
+
+
+class GitLabResourceGroup(GitLabProjects):
+    def get_specific_resource_group(self, project_and_group_name, resource_group_name):
+        return self._make_requests_to_api(
+            "projects/%s/resource_groups/%s",
+            (project_and_group_name, resource_group_name),
+            method="GET",
+            expected_codes=[200, 404],
+        )
+
+    def update_resource_group(
```

Comment:
It would be nice if this update function works similar to other update/put functions. That way if gitlab adds new feature and that is manged through additional attributes to this api, we can use it right away and don't have to modify gitlabform. What do you think? I'm not sure if the implementation does that already. It seems to have current known attribute, process_mode, hard coded. Please correct me if I'm wrong.

Reply:
After going through the documentation, I do not think that is possible. 

`PUT /projects/:id` vs. `PUT /projects/:id/resource_groups/:key`

We're going to have to hardcode `resource_groups` somewhere to append it to the URL and we need to specify the resource group name as well to determine which resource group's process mode to modify.

Reply:
I think let's chat @gdubicki  as I'm not fully familiar with the codebase. Seems the guideline is to allow raw parameter passing ability, which is very useful. Since it's a separate endpoint, maybe some additional setup is needed?